### ISSUE TITLE
2.0.1 fails to install on Ubuntu 16.04

### ISSUE THREAD

**Issue by Kautenja** [Kautenja/nes-py#39]
### Describe the bug

the package fails to install from pip on a machine running Ubuntu 16.04 with appropriate C++ libraries

### To Reproduce

1.  ensure nes-py is not installed
2. run `python3 -m pip install nes-py==2.0.1`
3. witness install failure

### Expected behavior

Install for use

### Environment

-   Operating System: Ubuntu 16.04
-   Python version: 3.5.2
-   C++ compiler and version: clang
-   `nes-gym` version: ?

### Additional context

2.0.0 will install successfully

**Comment by bic4907** [comment]
```
   running build_ext
    building 'nes_py.lib_nes_env' extension
    creating build/temp.linux-x86_64-3.5
    creating build/temp.linux-x86_64-3.5/nes_py
    creating build/temp.linux-x86_64-3.5/nes_py/cpp
    creating build/temp.linux-x86_64-3.5/nes_py/cpp/mappers
    x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -Wstrict-prototypes -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Ines_py/cpp/include -I/home/inchang/Desktop/mario_test/venv/include -I/usr/include/python3.5m -c nes_py/cpp/gui.cpp -o build/temp.linux-x86_64-3.5/nes_py/cpp/gui.o -std=c++1y -O2 -march=native -pipe
    cc1plus: warning: command line option ‘-Wstrict-prototypes’ is valid for C/ObjC but not for C++
    nes_py/cpp/gui.cpp:1:19: fatal error: gui.hpp: 그런 파일이나 디렉터리가 없습니다 (no such file or directory)
    compilation terminated.
    error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
    
    ----------------------------------------
Command "/home/inchang/Desktop/mario_test/venv/bin/python3 -u -c "import setuptools, tokenize;__file__='/tmp/pip-install-me6dbxp2/nes-py/setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record /tmp/pip-record-cclizqdv/install-record.txt --single-version-externally-managed --compile --install-headers /home/inchang/Desktop/mario_test/venv/include/site/python3.5/nes-py" failed with error code 1 in /tmp/pip-install-me6dbxp2/nes-py/
```
I have same installation error in this version(2.0.1).
It may be an error that occurred because the `gui.hpp` file was missing, but the file could not be found in previous versions.

**Comment by Kautenja** [comment]
ahh. i believe i know what the problem is now. I had changed the name of the directory with the CPP source files, but forgot to update the MANIFEST.in file so when I released the code to PyPi the CPP didn't get included. I'll be travelling today, but I'll have that fixed tonight. Sorry for any confusion.

**Comment by Kautenja** [comment]
issue resolved in version 2.0.2
